package.json

```diff
@@ -31,7 +32,7 @@
     "testcafe-reporter-slack": "^0.0.6"
   },
   "devDependencies": {
-    "@internetarchive/bookreader": "^5.0.0-38",
+    "@internetarchive/bookreader": "^5.0.0-49",
```

Comment:
QUESTION: Is it desired to have hard-coded versions here? Does that mean that we'll have to update version numbers in the e2e repo whenever we change package versions? Would it be better to always test the latest instead, so these versions don't need to be constantly updated? Or maybe I'm not understanding...

Reply:
i will look into pointing to `latest` tag...